README.md

```diff
@@ -133,6 +133,8 @@ Additional lists you might find useful:
 ## E-commerce
 *Plugins and applications for taking payments and building online e-commerce stores.*
 
+- [Paypal WPP plugin](https://github.com/cpierce/paypal-wpp) - Paypal Web Payments Pro Plugin for CakePHP
```

Comment:
Plugin name is PaypalWPP
A few tests would be good and probably a bit of docs so people know how to use it.